src/backend/vulkan/src/lib.rs

```diff
@@ -559,12 +569,51 @@ impl hal::Instance<Backend> for Instance {
                 None
             }
         };
+        #[cfg(feature = "renderdoc")]
+        let render_doc_entry = unsafe {
+            type GetApiFn =
+                unsafe extern "C" fn(version: u32, out: *mut *mut std::ffi::c_void) -> i32;
+
+            let renderdoc_filename = "renderdoc.dll";
+            let renderdorc_lib = libloading::Library::new(renderdoc_filename).map_err(|e| {
```

Comment:
We  shouldn't error on this. If we remove the feature, this code should be expected to handle *any* connection errors.
Let's also make it so this work is only done `if cfg!(debug_assertions)`.

Reply:
Using an `Option::None` here now instead. I was wondering whether a `Result<_, String>` might make sense, in order to log a warning when frame capture functionality is used but not available because of some initialization issue

Regarding `if cfg!(debug_assertions)`: Wouldn't that complete disable this functionality in production builds? I was under the impression that frame debugging was fairly useful on production code as well, and the overhead should be relatively minor.